shell/components/SortableTable/index.vue

```diff
@@ -294,17 +307,39 @@ export default {
     componentTestid: {
       type:    String,
       default: 'sortable-table'
-    }
+    },
+    hasAdvancedFiltering: {
```

Comment:
These three props should have some jsdoc comments. We haven't done a good job of this in the past but should going forward

Reply:
moved to the new mixin